e2e-tests/cypress/integration/common/actions.js

```diff
@@ -1,11 +1,11 @@
 import {Given, When} from "cypress-cucumber-preprocessor/steps";
 
-Given('I am registering as an organisation', () => {
+Given('I am registering as an {string}', (radioChoice) => {
```

Comment:
I think I prefer to not parameterise this step and instead use something more succinct. Let's add another 'Given` step with text "I am registering as an agent" or "I am registering on behalf of another party"